components/Icon/Icon.jsx

```diff
@@ -121,6 +123,7 @@ const Icon = ({ name, skin, size, ...props }) => {
     arrow_drop_up: ArrowDropUp,
     arrow_left: ArrowLeft,
     arrow_right: ArrowRight,
+    arrow_up: ArrowUpward,
```

Comment:
```suggestion
    arrow_upward: ArrowUpward,
```
Leaving the name exactly as per Material Design helps when searching for items with many variations.